docs/documentation.md

```diff
@@ -0,0 +1,35 @@
+# Getting Started
```

Comment:
Went with the latter option because it's less work.  I think the former option is better but can be done later.